src/ucp/wireup/select.c

```diff
@@ -1729,9 +1760,12 @@ ucp_wireup_add_am_bw_lanes(const ucp_wireup_select_params_t *select_params,
         }
     }
 
-    num_am_bw_lanes = ucp_wireup_add_bw_lanes(select_params, &bw_info,
-                                              ucp_tl_bitmap_max, am_lane,
-                                              select_ctx, 0);
+    ucp_wireup_dev_usage_count_init(&dev_count);
+    dev_count.excl_lane = am_lane;
+    bw_info.dev_count   = &dev_count;
```

Comment:
pls move these 3 lines right before line 1740

Reply:
Done. But had to move up am_lane finding logic as well.